microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,184 @@ ms.date: 05/04/2021
-    | `scripts` | A directory. |
+    | `microsoftedge.github.io` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `buttons.js` file is [https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js](https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js). |
+    | `css`, `photos` | Directories |
     | `(index)` | The main HTML document. |
-    | `sw.js` | A service worker runtime context. |
+    | `buttons.js` | A script to handle button presses on the page. |
 
-1. Select a resource to view it in the **Editor**.
+1. Select a resource file to view it in the Editor of the **Sources** tool:
 
-   ![View a file in the Editor.](../media/resources-sources-page-resource.msft.png)
+   ![Viewing a resource file in the Editor of the Sources tool](../media/resources-sources-page-resource.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse resources by filename in the Sources tool
+## Browse resource files sorted by filename in the Page tab of the Sources tool
+
+By default, the **Page** tab in the **Sources** tool groups resource files by folder.  To instead display all the resource files for each domain grouped together in a single alphabetized list:
 
-By default, the **Page** tab groups resources by directory.  To display the resources for each domain as a flat list, instead of grouping them by directory:
+1. In a new window or tab, go to a webpage, such as the [Accessibility-testing features demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/).  (The source files for that webpage are at [MicrosoftEdge / Demos > devtools-a11y-testing](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing).)
 
-1. Go to the **Sources** tool.
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-1. In the **Navigator** pane (on the left), select the **Page** tab.
+1. In DevTools, on the main toolbar, select the **Sources** tab.  If that tab isn't visible, click the **More tabs** (![The More tabs icon](../media/more-tabs-icon-light-theme.png)) button.
 
-1. Click **More options** `...` and then clear the **Group by folder** checkbox.
+1. In the **Navigator** pane in the upper left, select the **Page** tab.
 
-   ![The Group by folder option.](../media/resources-sources-page-resource-group-by-folder.msft.png)
+1. Click **More options** (...) and then clear the **Group by folder** option:
 
-    Resources are organized by file type.  Within each file type, the resources are organized alphabetically.
+   ![The Group by folder option in the Page tab of the Sources tool](../media/resources-sources-page-resource-group-by-folder.msft.png)
```

Comment:
add add'l red boxes per my design approach "in DevTools screenshots always highlight which tab & sub-tab" as basic elementary navigation & tool (tab) identification emphasis:
* Sources tab
* Page tab